### ISSUE TITLE
FaceVoiceModel - image_encoder 학습 비활성화

### ISSUE THREAD

**Issue by znxkznxk1030** [AI-Learning-Crew/LC_Pytorch#4]
**문제 설명:**
Pretrained image encoder를 학습 과정에 적용할 경우 모델의 수렴 속도와 안정성에 부정적인 영향을 미칠 수 있어, 해당 문제를 해결하고자 합니다.

**해결 방향:**

1. 학습률 스케줄링(Learning Rate Scheduling) 기법을 조정하여 진동을 완화할 수 있는 방안 모색
2. Pretrained encoder의 가중치를 고정(Fine-tuning 시 Freeze)하거나, 초기 학습률을 낮춰 점진적으로 조정하는 실험 진행
3. Gradient Clipping 적용을 통해 갑작스러운 가중치 변화로 인한 학습률 변동 방지

**예시 코드:**
```python

class FaceVoiceModel(nn.Module):
    def __init__(self, ...):
        super(FaceVoiceModel, self).__init__()
        # 이미지 인코더 (ViT)
        self.image_encoder = ViTModel.from_pretrained(image_model_name)
        
        # 이미지 인코더 파라미터 고정
        for param in self.image_encoder.parameters():
            param.requires_grad = False  # 학습 비활성화
        
        self.image_projection = nn.Linear(...)
        ...

```
